arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/SloadSection.java

```diff
@@ -113,8 +115,7 @@ public void resolveUponRollback(Hub hub, MessageFrame messageFrame, CallFrame ca
     // TODO: make sure that the "current" execution context is the one that is being rolled back
     //  so that we can use its revert stamp ()
     final DomSubStampsSubFragment undoingDomSubStamps =
-        DomSubStampsSubFragment.revertWithCurrentDomSubStamps(
-            this.hubStamp(), hub.callStack().currentCallFrame().revertStamp(), 0);
+        DomSubStampsSubFragment.revertWithCurrentDomSubStamps(hubStamp, callFrame.revertStamp(), 0);
```

Comment:
Fixed two issues for `SLOAD`:
- correct `HUB_STAMP`
- using the right call frame to extract the `REVERT_STAMP`